glide.yaml

```diff
@@ -13,3 +13,8 @@ import:
   version: kubernetes-1.13.4
 - package: k8s.io/api
   version: kubernetes-1.13.4
+- package: k8s.io/kubelet
+  repo:    git@github.com:openshift/kubernetes-kubelet
```

Comment:
why our fork?

Reply:
will change that back to upstream, as originally done in this PR.